src/common/util/firebase.ts

```diff
@@ -0,0 +1,217 @@
+import * as fb from 'firebase';
```

Comment:
Wonder why GitHub sometimes detects these as renamed files and sometimes doesn't 🤔